src/components/Perils/PerilList/PerilList.tsx

```diff
@@ -0,0 +1,63 @@
+import styled from '@emotion/styled'
+import React from 'react'
+import { LAPTOP_BP_UP, TABLET_BP_UP } from 'components/blockHelpers'
+import { PerilItem } from 'components/Perils/PerilItem/PerilItem'
+import { minimalColorComponentColors } from 'src/blocks/BaseBlockProps'
```

Comment:
nit: should this import be updated?

```suggestion
import { minimalColorComponentColors } from 'blocks/BaseBlockProps'
```